LandLord-core/src/main/java/biz/princeps/landlord/placeholderapi/LLExpansion.java

```diff
@@ -33,75 +49,104 @@ public String getVersion() {
+                                    "player has finished! Use FinishedLoadingPlayerEvent!");
+                            return "NaN";
+                        }
+                        return String.valueOf(iPlayer2.getClaims() - wg.getRegionCount(player.getUniqueId()));
+
+                    case "current_land_owner":
+                        final IOwnedLand region = wg.getRegion(player.getLocation());
+                        if (region != null && region.getOwner() != null) {
+                            return region.getOwnersString();
+                        }
+                        return "∅";
+
+                    case "current_land_members":
+                        final String members;
+                        final IOwnedLand region2 = wg.getRegion(player.getLocation());
+                        if (region2 != null && !(members = region2.getMembersString()).isEmpty()) {
+                            return members;
+                        }
+                        return "∅";
+
+                    case "current_land_name":
+                        return wg.getLandName(player.getLocation().getChunk());
+
+                    case "next_land_price":
+                        return String.valueOf(pl.getCostManager().calculateCost(player.getUniqueId()));
+
+                    case "current_land_refund":
+                        final int regionCount = wg.getRegionCount(player.getUniqueId());
+                        return String.valueOf(pl.getCostManager().calculateCost(regionCount - 1) * pl.getConfig().getDouble(
+                                "Payback"));
+
+                    case "max_claim_permission":
+                        return String.valueOf(getMaxClaimPermission(player));
+
+                    case "remaining_free_lands":
+                        final int landCount = wg.getRegionCount(player.getUniqueId());
+                        final int freeLands = pl.getConfig().getInt("Freelands");
+
+                        if (landCount <= freeLands) {
+                            return String.valueOf((Math.min(getMaxClaimPermission(player), freeLands)) - landCount);
+                        } else {
+                            return "0";
+                        }
+
+                    default:
+                        return null;
                 }
+            });
+        } catch (ExecutionException e) {
+            pl.getLogger().warning("Could not parse placeholder: " + placeholder + " for " + player.getName() + "!");
+            e.printStackTrace();
+        }
+        return null;
+    }
 
-            default:
-                return null;
+    private int getMaxClaimPermission(Player player) {
+        try {
+            return maxClaimPermissionCache.get(player.getUniqueId(), () ->
+                    pl.getPlayerManager().getMaxClaimPermission(player));
+        } catch (ExecutionException e) {
+            pl.getLogger().warning("Could not get maxClaimPermission for " + player.getName() + "!");
```

Comment:
Use the logger to log the exception as well.
`pl.getLogger().log(Level.SEVERE, "Could not get maxClaimPermission for " + player.getName() + "!", e);`